src/app_service/service/accounts/service.nim

```diff
@@ -46,6 +49,53 @@ proc getImportedAccount*(self: Service): GeneratedAccountDto =
 proc isFirstTimeAccountLogin*(self: Service): bool =
   return self.isFirstTimeAccountLogin
 
+# Remove extra copied keystore that are not needed (can be remove 1st jan 2023)
+proc deleteExtraKeyStoreFile*(self: Service) =
+  let accounts = status_account.getAccounts().result
+  var deleteCandidates: seq[string] = @[]
+  for path in walkFiles(self.keyStoreDir & "*"):
+    var found = false
+    for account in accounts.getElems():
+      let address = account{"address"}.getStr.replace("0x", "")
+      if path.endsWith(address):
```

Comment:
@alaibe are the files under the `keyStoreDir` always without extension on all platforms? I checked on mac, and they are, but not sure about windows and linux, if so than all good, if not maybe `contains` would be better than `endsWith`.

Reply:
On linux too they are, good point about Windows (i think they are too). Jo will also test the PR he can confirm or not?